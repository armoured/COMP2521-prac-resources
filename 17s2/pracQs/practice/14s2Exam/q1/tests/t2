./q1 mul m0 m0
