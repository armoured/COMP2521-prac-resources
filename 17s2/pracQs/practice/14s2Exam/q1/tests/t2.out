Matrix X
[     1     2 ]
[     3     4 ]
Matrix Y
[     1     2 ]
[     3     4 ]
multiply(X,Y)
[     7    10 ]
[    15    22 ]
